packages/xo-web/src/common/intl/messages.js

```diff
@@ -570,7 +570,8 @@ const messages = {
   newSrCreate: 'Create',
   newSrNamePlaceHolder: 'Storage name',
   newSrDescPlaceHolder: 'Storage description',
-  newSrAddressPlaceHolder: 'Address',
+  newSrAddressPlaceHolder: 'IP Address',
```

Comment:
IMHO, this placeholder is useless (does not provide any new info) and should removed in favor of explicit placeholder by SR type.